### PR TITLE
FEATURE: Add flow annotations to attribute set

### PR DESCRIPTION
**Pull request by saschanowak**
With this set your flow code can easily migrated
to a modern php8 codebase.

### CODE REVIEW

File: config/sets/flow-annotations-to-attributes.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        new AnnotationToAttribute('Neos\\Flow\\Annotations\\Pointcut'),
+        new AnnotationToAttribute('Neos\\Flow\\Annotations\\Proxy'),
+        new AnnotationToAttribute('Neos\\Flow\\Annotations\\Scope'),
+        new AnnotationToAttribute('Neos\\Flow\\Annotations\\Session'),
+        new AnnotationToAttribute('Neos\\Flow\\Annotations\\Signal'),
+        new AnnotationToAttribute('Neos\\Flow\\Annotations\\SkipCsrfProtection'),
+        new AnnotationToAttribute('Neos\\Flow\\Annotations\\Transient'),
+        new AnnotationToAttribute('Neos\\Flow\\Annotations\\Validate'),
+        new AnnotationToAttribute('Neos\\Flow\\Annotations\\ValidationGroups'),
+        new AnnotationToAttribute('Neos\\Flow\\Annotations\\ValueObject'),
```
Comment (bwaidelich) [inline config/sets/flow-annotations-to-attributes.php:37]: Any reason for not using `::class`-constants here for greater type safety?

**Review by bwaidelich — APPROVED** [review]
Lovely, thanks a lot!
Just one question re class strings, otherwise +1 (by reading)

File: config/sets/flow-annotations-to-attributes.php (hunk 1 above)
Comment (bwaidelich) [reply config/sets/flow-annotations-to-attributes.php:37]: > Switching to ::class would show a lot of warnings in the IDE as these classes are not present in the code

That's true and for the same reason there is no type safety anyways. I'd probably leave it as is

**Comment by dlubitz** [comment]
@bwaidelich What about merging this? 

Looks good to me, except of the move of the set files. I'd keep it in `/config/set` (singular) as rector is doing it the same way (https://github.com/rectorphp/rector/tree/main/config/set)

**Comment by bwaidelich** [comment]
It has enough votes but merge conflicts

**Review by dlubitz — APPROVED** [review]

**Comment by dlubitz** [comment]
Thanks @saschanowak for this contribution. 👏

**Pull request merged**